bookwyrm/redis_store.py

```diff
@@ -4,12 +4,7 @@
 
 from bookwyrm import settings
 
-r = redis.Redis(
-    host=settings.REDIS_ACTIVITY_HOST,
-    port=settings.REDIS_ACTIVITY_PORT,
-    password=settings.REDIS_ACTIVITY_PASSWORD,
-    db=settings.REDIS_ACTIVITY_DB_INDEX,
-)
+r = redis.from_url(settings.REDIS_ACTIVITY_URL)
```

Comment:
will redis connect correctly to the broker in [bookwyrm/tasks.py](https://github.com/bookwyrm-social/bookwyrm/blob/main/bookwyrm/tasks.py#L10) using the `CELERY_BROKER_URL` given the way you're using the `REDIS_` url here?

Reply:
Yes. The idea is that you'd use `unix://` in your env for socket urls, since that format is used everywhere else, and the url is changed `redis+socket://` for the variables used by celery